packages/turbo/ios/RNVisitableView.swift

```diff
@@ -14,14 +14,23 @@ let REFRESH_SCRIPT = "typeof Turbo.session.refresh === 'function'" +
 class RNVisitableView: UIView, RNSessionSubscriber {
   var id: UUID = UUID()
   @objc var sessionHandle: NSString? = nil
-  @objc var applicationNameForUserAgent: NSString? = nil
   @objc var url: NSString = "" {
     didSet {
       if(url != oldValue) {
         visit()
       }
     }
   }
+  @objc var applicationNameForUserAgent: NSString? = nil {
+    didSet {
+      sessionConfiguration.webViewConfiguration.applicationNameForUserAgent = applicationNameForUserAgent as? String
```

Comment:
If the pattern with the configurator should be there, I think _it_ should have a `applicationNameForUserAgent` property that we set directly here. It feels a bit off to access (and manipulate) the deeply nested `applicationNameForUserAgent` and it breaks the pattern of having the configurator object.

Eg. something like:

```swift
didSet {
  sessionConfiguration.applicationNameForUserAgent =  ..
}
```

and then in the configurator:

```swift
var applicationNameForUserAgent {
  didSet {
    webViewConfiguration.applicationNameForUserAgent = applicationNameForUserAgent
  }
}
```